doc/tutorial/faq_tutorial.txt

```diff
@@ -64,7 +64,7 @@ Defining the theano function
 
 .. code-block:: python
 
-   f = theano.function(..., updates=updates)
```

Comment:
This isn't new. It is just new doc for a common case. lookup_table is a matrix shared variable defined at line 26.